### PR TITLE
Add ActiveIssue Annotations to Failing TestReadyToRun Tests and Reenable the Executables

### LINKED ISSUE #95928: Enable Remaining Libraries Tests in TestReadyToRun Pipeline
PR #91229 adds a new pipeline where we build and run the libraries tests with R2R enabled. We spent a lot of time trying to get them all running at once, but it proved to be a neverending task due to the amount of hidden test cases that would fail and crash/halt the entire test runs. So, in order to get the benefits of the coverage these new pipelines provide, we only enabled the assemblies we confirmed worked and passed correctly. Which leads us to the creation of this issue. This is a tracking issue of the necessary work to gradually enable the remaining tests.

### PR DESCRIPTION
**Pull request by ivdiazsa**
Completes #95928. This PR will introduce the necessary changes to enable as many libraries test executables as we can for the `TestReadyToRun` pipeline.

### CODE REVIEW

**Comment by ivdiazsa** [comment]
I'm still in the process of filing the failing tests' issues. Will update the `ActiveIssue` template annotations as I get them.

File: src/libraries/System.Console/tests/WindowAndCursorProps.cs [hunk 1]
```diff
@@ -119,6 +119,7 @@ public static void WindowHeight_SetUnix_ThrowsPlatformNotSupportedException()
 
     [Fact]
     [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser & ~TestPlatforms.iOS & ~TestPlatforms.MacCatalyst & ~TestPlatforms.tvOS)]  // Expected behavior specific to Unix
+    // [ActiveIssue("Issue", typeof(PlatformDetection), nameof(PlatformDetection.IsReadyToRunCompiled))]
```
Comment (ivdiazsa) [inline src/libraries/System.Console/tests/WindowAndCursorProps.cs:122]: Wait why is this here? I'll check whether we need this or not. Working on it.

**Review by trylek — APPROVED** [review]
LGTM once you figure out what's up with the commented out ActiveIssue attribute.

**Comment by ivdiazsa** [comment]
Failure is unrelated, so merging this now.

**Pull request merged**